beacon-chain/blockchain/receive_block_test.go

```diff
@@ -108,6 +109,9 @@ func TestService_ReceiveBlock(t *testing.T) {
 				block: genFullBlock(t, util.DefaultBlockGenConfig(), 1 /*slot*/),
 			},
 			check: func(t *testing.T, s *Service) {
+				// Hacky sleep, should use a better way to be able to resolve the race
+				// between event being sent out and processed.
+				time.Sleep(100 * time.Millisecond)
```

Comment:
I wonder if there's a way to add a mock or to skip something to isolate some of this to skip this hacky sleep. at the very least maybe adding a TODO marker will keep track of it.